calendar_reboot.js

```diff
@@ -66,44 +65,54 @@ $(document).ready(function() {
         //with the class form-group, also adds the hidden class on the span
         //with class sr-only (for screen readers)
         
-        id.removeClass('has-error has-feedback');
-        srId.removeClass('hidden');
+        $(id).removeClass('has-error has-feedback');
+        $(srId).removeClass('hidden');
     };
     
     var addGlyphicon = function(id) {
         //removes the hidden class to the glyphicon tag with id = id
-        id.removeClass('hidden');
+        $(id).removeClass('hidden');
     };
     
     var removeGlyphicon = function(id) {
         //adds the hidden class to the glyphicon tag with id=id
-        id.addClass('hidden');
+        $(id).addClass('hidden');
     };
     
-    validateForm = function(startDate, endDate) {
-        //check that endDate comes after startDate
-        //checks 
-          //if there is no valid startDate, prompt user for a valid startDate
+    var removeFormErrors = function() {
+        //remove the error classes and glyphicons from the form inputs
+        removeFormError('#dateFormGroup', '#inputError-dateFormGroup');
+        removeFormError('#dateFormGroup2', '#inputError-dateFormGroup2');
+        removeFormError('#titleFormGroup', '#inputError-titleFormGroup');
+        removeGlyphicon('#span-titleFormGroup');
+    };
+    
+    var validateDates = function(startDate, endDate) {
```

Comment:
I changed it to xxxDateString. Elsewhere in the code, xxxDateString is formatted "YYYYMMDD" but here it is "YYYY-MM-DD" because this function is taking the string that is obtained from the datetimepicker. Is that a problem?